ci/Jenkinsfiles/build.groovy

```diff
@@ -320,12 +320,11 @@ pipeline {
               boolean isReferenceBranch = BRANCH_NAME == REFERENCE_BRANCH
               // first substitute docker image names and versions
               withCredentials([usernamePassword(credentialsId: 'explorer-preview-maintenance', passwordVariable: 'EXPLORER_PASSWORD', usernameVariable: 'UNUSED_USERNAME')]) {
-                def explorerPasswordProp = "org.nuxeo.apidoc.apidocAdmin.password=${EXPLORER_PASSWORD}"
-                sh """
+                sh '''
                   mv values.yaml values.yaml.tosubst
-                  NUXEO_EXPLORER_CUSTOM_PARAMS=${isReferenceBranch ? explorerPasswordProp : ''} \
+                  NUXEO_EXPLORER_CUSTOM_PARAMS=${isReferenceBranch ? 'org.nuxeo.apidoc.apidocAdmin.password=$EXPLORER_PASSWORD' : ''} \
```

Comment:
This one fails on master with:
```
line 3: ${isReferenceBranch ? 'org.nuxeo.apidoc.****.password=$EXPLORER_PASSWORD' : ''}: bad substitution
```

Reply:
Yes, same issue as https://github.com/nuxeo/nuxeo-explorer/pull/105/files#r584718096 probably.
I think you can try `"org.nuxeo.apidoc.apidocAdmin.password=${EXPLORER_PASSWORD}"` to have the substitution.

Reply:
Actually sorry i do have a warn on EXPLORER_PASSWORD so maybe i should do the following change:
Replace
```
def explorerPasswordProp = "org.nuxeo.apidoc.apidocAdmin.password=${EXPLORER_PASSWORD}"
```
By
```
def explorerPasswordProp = "org.nuxeo.apidoc.apidocAdmin.password=$EXPLORER_PASSWORD"
```

?

Reply:
We want to get rid of Groovy interpolation, aka "${SENSITIVE NEV VAR}" or """ ${SENSITIVE NEV VAR} """, meaning, we cannot have sensitive environment variables manipulated in a Groovy script...

So, in this case, I guess the simpler would be:
```
          sh 'mv values.yaml values.yaml.tosubst'
          if (isReferenceBranch) {
            withCredentials([usernamePassword(credentialsId: 'explorer-preview-maintenance', passwordVariable: 'EXPLORER_PASSWORD', usernameVariable: 'UNUSED_USERNAME')]) {
              sh '''
                NUXEO_EXPLORER_CUSTOM_PARAMS=org.nuxeo.apidoc.apidocAdmin.password=$EXPLORER_PASSWORD \
                  envsubst < values.yaml.tosubst > values.yaml
              '''
            }
          } else {
            sh 'envsubst < values.yaml.tosubst > values.yaml'
          }
```